examples/js_of_ocaml/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>Js_of_ocaml • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+	</head>
+	<body>
+		<div id="todomvc"></div>
```

Comment:
Could we have an `id` attribute and a `class` attribute? ie something like:
```html
<div id="todomvc" class="todoapp"></div>
```

Otherwise, if the idea is to ban any `id` attribute, i have no solution for now until the next release of js_of_ocaml (which will include the `getElementsByClassName` method).

By the way, can you confirm that the needed class name is `todoapp`? Thanks.